### ISSUE TITLE
docker containers not comming up

### ISSUE THREAD

**Issue by 1day2die** [onetimesecret/onetimesecret#260]
Using docker compose on Ubuntu , the containers do not seem to start . No errors, no Logs
Using CURL i get a "connection refused" on port 7143 and 3000

![image](https://github.com/onetimesecret/onetimesecret/assets/8725848/0d89174a-5a1c-462a-9916-00b8fd7ee0ec)

![image](https://github.com/onetimesecret/onetimesecret/assets/8725848/3b853a6d-bd68-447f-b354-24f032a5bcfe)

**Comment by emilyHere6** [comment]
@1day2die Did you ever get this worked out? I'm having a similar issue.

**Comment by 1day2die** [comment]
nope. sadly not

**Comment by Vasilevs2** [comment]
Could you share your Config and .env files?

**Comment by emilyHere6** [comment]
@1day2die @Vasilevs2 Yeah maybe if we all share our config files they can combine to work haha
